src/main.ts

```diff
@@ -75,10 +76,9 @@ start().catch(error => {
     alert('There was an error starting this page. (See console for details.)\n' +
       'Please try again later.')
   }
-  // try to redirect to Business Registry home page
-  const businessesUrl = sessionStorage.getItem('BUSINESSES_URL')
-  if (businessesUrl) {
-    // assume Businesses URL is always reachable
-    window.location.assign(businessesUrl)
+  // try to redirect to BCROS home page
+  const bcrosHomeUrl = sessionStorage.getItem('BUSINESSES_URL')
```

Comment:
Doesn't have to be here or now, but would it make sense to assign these gets from the session to a const where we can just import the const file. Might save having to re-define these individually. 
Just a random idea, no  action required!

Reply:
Well actually... I did try to put these in a mixin as getters, which works for _most_ of the properties...

Keycloak Token didn't get updated (prop because the built-in Vue watcher didn't recognize the update, which was done by a different task).

And some modules (main.ts, router.ts) aren't Vue classes and couldn't use the mixin.

So, to limit the amount of code y'all have to review, I undid my changes (but kept them as local getters for cleaner code).

Reply:
Totally, that makes perfect sense!